src/panel/poi_bloc/PoiBlockContainer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import HourBlock from '../../views/poi_bloc/Hour';
+import ContactBlock from '../../views/poi_bloc/Contact';
+import ImagesBlock from '../../views/poi_bloc/Images';
+import WebsiteBlock from '../../views/poi_bloc/Website';
+import InformationBlock from '../../views/poi_bloc/Information';
+
+export default class PoiBlockContainer extends React.Component {
+  static propTypes = {
+    poi: PropTypes.object,
+  }
+
+  constructor(props) {
```

Comment:
This is not needed if you don't do anything else than `super(props)` in it, as it's the default behavior when there is no explicit constructor.